### ISSUE TITLE
Replace/remove existing logic

### ISSUE THREAD

**Issue by larsbo** [shopperlabs/framework#63]
Hi, nice project!
Is it possible to replace/remove existing controllers / live wire components or routes e.g. to use this as a marketplace by adding additional logic?

**Comment by nurmuhammet-ali** [comment]
@larsbo fork it, and play. Or if just directly edit in vendor package.  Don't recommend it, but that could be the solution.

**Comment by mckenziearts** [comment]
Yes @larsbo it's possible. i will finish the documentation soon, you can extend shopper or replace existing features by your own if you don't like the current implementation. And Shopper is build using component you can easily reuse to create your own design

**Comment by cdigruttola** [comment]
Hello,
where can I find the documentation to extedn or replace existing features?
Thanks
